servicetalk-loadbalancer-experimental-provider/src/main/java/io/servicetalk/loadbalancer/experimental/DefaultLoadBalancerObserver.java

```diff
@@ -59,6 +59,19 @@ public void onNoActiveHostsAvailable(int hostSetSize, NoActiveHostException exce
         LOGGER.debug("{}- No active hosts available. Host set size: {}.", lbDescription, hostSetSize, exception);
     }
 
+    @Override
+    public void onHostSetChanged(Collection<? extends Host> newHosts) {
+        if (LOGGER.isDebugEnabled()) {
+            int healthyCount = 0;
+            for (Host host : newHosts) {
+                if (host.isHealthy()) {
+                    healthyCount++;
+                }
+            }
+            LOGGER.debug("{}- onHostSetChanged(total: {}, healthy: {})", lbDescription, newHosts.size(), healthyCount);
```

Comment:
Will it be helpful to log entire `Collection` as well?

Reply:
At debug I suppose it won't hurt.